msdk-featdet/pom.xml

```diff
@@ -15,5 +15,6 @@
         <module>msdk-featdet-gridmass</module>

```

Comment:
Please also add a dependency to the msdk-all module, which aggregates all MSDK modules.